tests/first_stage_test.py

```diff
@@ -1,11 +1,15 @@
+import fcntl
+import os
+import select
 import subprocess
+import sys
+
+import testlib
```

Comment:
Style and git cleanliness nit: did you mean to move this? Why?

My general preferences
- testlib is a local import (same directory), that's why it was last (sfter stdlib, third party, and packages) ala https://peps.python.org/pep-0008/#imports
- don't change lines unnecessarily, to avoid complicating `git annotate` output.

Reply:
Hmm, that was done by ruff, but yes I agree, it doesn't make any sense here to move.